### PR TITLE
docs: add website content

### PR DESCRIPTION
**Pull request by efekarakus**
Add `site/` directory to hold the content for our the copilot website.

_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

**Comment by efekarakus** [comment]
You can preview the website here https://efekarakus.github.io/copilot-demo-docs/

**Comment by efekarakus** [comment]
Maybe i should ignore the theme and instead pull the submodule every time other the changes are hugee

**Comment by kohidave** [comment]
> Maybe i should ignore the theme and instead pull the submodule every time other the changes are hugee

you thumbed up yourself...

File: site/content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+Installing the AWS Copilot CLI currently requires you to download our binary from the GitHub releases page manually. In the future, we'll distribute the binary through homebrew and other binaries as well.
+
+In the meantime, to install, copy and paste the command into your terminal.
+
+| Platform | Command to install |
+|---------|---------
+| macOS | `curl -Lo /usr/local/bin/copilot https://github.com/aws/copilot-cli/releases/download/v0.1.0/copilot-darwin-v0.1.0 && chmod +x /usr/local/bin/copilot && copilot --help` |
+| Linux | `curl -Lo /usr/local/bin/copilot https://github.com/aws/copilot-cli/releases/download/v0.1.0/copilot-linux-v0.1.0 && chmod +x /usr/local/bin/copilot && copilot --help` |
```
Comment (iamhopaul123) [inline site/content/en/docs/Installation/_index.md:14]: How about our windows friends🤔

File: site/content/en/docs/Getting started/_index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+```bash
+$ copilot init
+```
+
+<img class="img-fluid" alt="gettingstarted" src="https://user-images.githubusercontent.com/879348/86040246-8d304400-b9f8-11ea-9590-2878c3a1d3de.png">
+
+#### Step 4: Answer a few questions
+
+The next thing we’re going to do is answer a few questions from Copilot. Copilot will use these questions to help us choose the best AWS infrastructure for your service. There’s only a few so let’s go through them:
```
Comment (iamhopaul123) [inline site/content/en/docs/Getting started/_index.md:43]: maybe "answering a few questions..."?

File: site/content/en/docs/Getting started/_index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+weight: 4
+---
+
+AWS Copilot makes it easy to deploy your containers to AWS in just a few steps. In this tutorial we’re going to do just that - we’re going to deploy a sample front end service that you can visit in your browser. While we’ll be using a sample static website in this example, you can use AWS Copilot to build and deploy any container app with a Dockerfile. After we get your service all set up, we’ll show you how to delete the resources Copilot created to avoid charges.
+
+Sound fun? Let’s do it!
+
+#### Step 1: Download & Configure AWS Copilot
+
+You’ll need a few things to use AWS Copilot - the AWS Copilot binary, AWS CLI, Docker Desktop and AWS credentials.
```
Comment (iamhopaul123) [inline site/content/en/docs/Getting started/_index.md:13]: Feel like a lot of potential customers will have problems that they don't have a well-configured AWS CLI and aws profile. Maybe we should put more details about it in our wiki/doc?

File: site/content/en/docs/Installation/_index.md (hunk 1 above)
Comment (efekarakus) [reply site/content/en/docs/Installation/_index.md:14]: I don't think we're listing the windows binary for now as we haven't tested the binary in a while :P

Comment (iamhopaul123) [reply site/content/en/docs/Installation/_index.md:14]: i wonder how many users are using Windows...maybe we should put windows support on our calendar if they are a considerable amount of them

File: site/content/en/docs/Getting started/_index.md (hunk 3 above)
Comment (efekarakus) [reply site/content/en/docs/Getting started/_index.md:13]: Discussed offline, we'll make updates in a follow-up PR along with some CSS changes

File: site/content/en/docs/Installation/_index.md (hunk 1 above)
Comment (RoniqueRicketts) [reply site/content/en/docs/Installation/_index.md:14]: I was wondering about the windows version of this cli. I am new to containers and the demo you guys did so far on how to push containers to ECS  is straightforward  but I couldn't find how to install  this for my win machine.

Comment (efekarakus) [reply site/content/en/docs/Installation/_index.md:14]: > I was wondering about the windows version of this cli. I am new to containers and the demo you guys did so far on how to push containers to ECS is straightforward but I couldn't find how to install this for my win machine.

Hi @RoniqueRicketts! We do provide a windows binary under our [releases](https://github.com/aws/copilot-cli/releases/tag/v0.1.0): [copilot-windows-v0.1.0.exe](https://github.com/aws/copilot-cli/releases/download/v0.1.0/copilot-windows-v0.1.0.exe) so you should be able to run the executable. However, we haven't tested the experience with the windows binary for a while. 

Feel free to open an issue if you experience difficulties with the binary!

**Pull request merged**